profiles/common/modules/features/nexteuropa_editorial/nexteuropa_editorial.install

```diff
@@ -91,7 +91,6 @@ function nexteuropa_editorial_install() {
   multisite_config_service('title')->replaceTitleField('node', 'editorial_team', 'title');
 
   // Add NextEuropa specific moderation states.
-  multisite_config_service('workbench_moderation')->createModerationState('validated', 'Validated', 'Ready to be reviewed', 0);
```

Comment:
Yes true, but we intend to remove the module multisite_rules_configuration which is quite useless.